News-Android-App/src/main/java/de/luhmer/owncloudnewsreader/NewsReaderListActivity.java

```diff
@@ -682,6 +684,42 @@ private NewsReaderDetailFragment updateDetailFragment(long id, Boolean folder, L
         return fragment;
     }
 
+	private void updateDetailFragmentTitle() {
```

Comment:
@proninyaroslav  Great idea to update the title if the title of the folder / feed changes. I'm wondering if it makes sense to combine the logic of this method with the logic in the `updateDetailFragment` method above. What do you think? I guess you probably already thought about it but decided to go with this approach, right? From what I can see your code adds a little more safety in terms of more null pointer checks.

Reply:
`updateDetailFragment` also affects the items list, which is not very logical, because further down the code we also update it.